src/app/initialisation/syncPoolService.js

```diff
@@ -43,14 +48,22 @@ function SyncPoolService($q, mediator, workorderSync, workflowSync, resultSync,
     promises.push(workflowSync.createManager());
     promises.push(messageSync.createManager());
     promises.push(resultSync.createManager());
-    return $q.all(promises).then(function(managers) {
-      var map = {};
-      managers.forEach(function(managerWrapper) {
-        map[managerWrapper.manager.datasetId] = managerWrapper;
+
+    //Initialising the sync managers for the required datasets.
+    return syncService.manage(config.datasetIds.workorders, {}, {}, config.syncOptions)
```

Comment:
[ProTip] I would do this by iterating by this set of ids. Having this done this way would allow us to work more framework way. So users would need to add new datasetId to config and it would be automatically managed by